test/WebJobs.Extensions.Tests/PublicSurfaceTests.cs

```diff
@@ -52,85 +52,83 @@ public void ExtensionsPublicSurface_LimitedToSpecificTypes()
             AssertPublicTypes(expected, assembly);
         }
 
-        [Fact]
-        public void SendGridPublicSurface_LimitedToSpecificTypes()
-        {
-            var assembly = typeof(SendGridAttribute).Assembly;
-
-            var expected = new[]
-            {
-                "SendGridAttribute",
-                "SendGridConfiguration",
-                "SendGridJobHostConfigurationExtensions"
-            };
-
-            AssertPublicTypes(expected, assembly);
-        }
-
-        [Fact]
-        public void MobileTablesPublicSurface_LimitedToSpecificTypes()
-        {
-            var assembly = typeof(MobileTableAttribute).Assembly;
-
-            var expected = new[]
-            {
-                "MobileTableAttribute",
-                "MobileAppsConfiguration",
-                "MobileAppsJobHostConfigurationExtensions"
-            };
-
-            AssertPublicTypes(expected, assembly);
-        }
-
-        [Fact]
-        public void DocumentDBPublicSurface_LimitedToSpecificTypes()
-        {
-            var assembly = typeof(DocumentDBAttribute).Assembly;
-
-            var expected = new[]
-            {
-                "DocumentDBAttribute",
-                "DocumentDBConfiguration",
-                "DocumentDBJobHostConfigurationExtensions",
-                "CosmosDBTriggerAttribute"
-            };
-
-            AssertPublicTypes(expected, assembly);
-        }
-
-        [Fact]
-        public void NotificationHubsPublicSurface_LimitedToSpecificTypes()
-        {
-            var assembly = typeof(NotificationHubAttribute).Assembly;
-
-            var expected = new[]
-            {
-                "NotificationHubAttribute",
-                "NotificationHubsConfiguration",
-                "NotificationHubJobHostConfigurationExtensions"
-            };
-
-            AssertPublicTypes(expected, assembly);
-        }
-
-        [Fact]
-        public void ApiHubPublicSurface_LimitedToSpecificTypes()
-        {
-            var assembly = typeof(ApiHubFileAttribute).Assembly;
-
-            var expected = new[]
-            {
-                "ApiHubFileAttribute",
-                "ApiHubFileTriggerAttribute",
-                "ApiHubConfiguration",
-                "ApiHubJobHostConfigurationExtensions",
-                "ApiHubTableAttribute",
-                "ConnectionFactory"
-            };
-
-            AssertPublicTypes(expected, assembly);
-        }
-
+        //[Fact]
```

Comment:
Here you're commenting out extension specific code, but other places you're deleting it. Let's use a common strategy